docs/source/markdown/podman-image-trust.1.md

```diff
@@ -54,14 +56,17 @@ Trust may be updated using the command **podman image trust set** for an existin
             registry scope
     **reject**: do not accept images for this registry scope
 
-## show OPTIONS
-
-#### **--raw**
-  Output trust policy file as raw JSON
+#### show OPTIONS
```

Comment:
```suggestion
### show OPTIONS
```

Reply:
Two was too big and conflicting with OPTIONS, but Four is too small and conflicts with the options themselves.  I'd go for in between with three hashes for this and `set OPTIONS`

Reply:
Fixed